src/main/java/org/elasql/storage/tx/concurrency/fifolocker/FifoLock.java

```diff
@@ -0,0 +1,48 @@
+package org.elasql.storage.tx.concurrency.fifolocker;
```

Comment:
I suggest to put this class back to `org.elasql.storage.tx.concurrency`.

Reply:
Roger that